test/e2erp/rotation.go

```diff
@@ -0,0 +1,141 @@
+package e2erp
```

Comment:
Rename this file to helpers.go or something more general since some of these functions are not specific to key rotation.